core/services/ocr2/plugins/functions/integration_tests/v1/functions_integration_test.go

```diff
@@ -27,33 +27,6 @@ func TestIntegration_Functions_MultipleV1Requests_Success(t *testing.T) {
 
 	utils.SetupRouterRoutes(t, b, owner, routerContract, active.Address, proposed.Address, allowListContractAddress)
 
-	_, _, oracleIdentities := utils.CreateFunctionsNodes(t, owner, b, routerAddress, nOracleNodes, maxGas, nil, nil)
```

Comment:
Why are you removing this? Can't we keep it with empty secrets?

Reply:
Yes. It will just require modifications elsewhere in the test as we currently check that secrets are present.